pkg/devcontainer/compose.go

```diff
@@ -435,6 +445,55 @@ func (r *Runner) buildAndExtendDockerCompose(parsedConfig *config.SubstitutedCon
 	return buildImageName, dockerComposeFilePath, imageMetadata, extendImageBuildInfo.MetadataLabel, nil
 }
 
+func (r *Runner) buildFeatureContentImage(ctx context.Context, featureBuildInfo *feature.BuildInfo) (string, error) {
+	helper, err := r.Driver.ComposeHelper()
+	if err != nil {
+		return "", err
+	}
+
+	tempFeatureContentImage := "dev_container_feature_content_temp"
```

Comment:
nit: could be a constant as it's not immediately clear this will be used by `getFeatureBuildOptions`